avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -542,9 +546,12 @@ def updatePositionC(cfg, particles, dem, force, DT):
   cdef double TotkinEne = particles['kineticEne']
   cdef double TotpotEne = particles['potentialEne']
   cdef double peakKinEne = particles['peakKinEne']
+  cdef double peakForceSPH = particles['peakForceSPH']
+  cdef double TotkForceSPH = particles['forceSPHIni']
   # initialize outputs
   cdef double TotkinEneNew = 0
   cdef double TotpotEneNew = 0
+  cdef double TotForceSPHNew = 0
```

Comment:
same here. Feel free to rename the others as well....

Reply:
done